templates/emails/oauth.application.authorized.hbs

```diff
@@ -0,0 +1,27 @@
+Subject: Open Collective: A third-party OAuth application has been added to your account
+
+{{> header}}
+
+<p>
+    <br />
+    Hi,
+    <br />
+    <br />
+    A third-party OAuth application ({{application.name}}) was recently authorized to access your account.
+    
+    {{#if scope.length}}
+    <br />
+    <br />
+    Scopes:
+    <br />
+    <ul>
```

Comment:
By HTML standards, flow elements shouldn't be used inside the paragraph element.
It is a good idea to move the list element to the root level of this file.

Reply:
Thanks for catching this, I updated it.